arithmetization/src/main/java/net/consensys/linea/zktracer/module/bin/Bin.java

```diff
@@ -34,11 +35,16 @@
 public class Bin implements Module {
   private static final byte NOT = (byte) 0x19;
 
+  private final Hub hub;
   private int stamp = 0;
 
   /** A set of the operations to trace */
   private final StackedSet<BinOperation> chunks = new StackedSet<>();
 
+  public Bin(final Hub hub) {
+    this.hub = hub;
+  }
```

Comment:
You can use `@RequiredArgumentsConstructor` here to avoid writing the constructor by hand.